src/main/java/uk/gov/hmcts/divorce/systemupdate/event/SystemPronounceCase.java

```diff
@@ -92,15 +92,20 @@ public AboutToStartOrSubmitResponse<CaseData, State> aboutToSubmit(CaseDetails<C
             updateMissingFields(caseData, user, serviceAuth);
         }
 
-        generateConditionalOrderGrantedDocs(details, beforeDetails);
-        notificationDispatcher.send(conditionalOrderPronouncedNotification, caseData, details.getId());
+        AboutToStartOrSubmitResponse.AboutToStartOrSubmitResponseBuilder<CaseData, State> responseBuilder =
+            AboutToStartOrSubmitResponse.builder();
 
-        final State state = caseData.isJudicialSeparationCase() ? SeparationOrderGranted : ConditionalOrderPronounced;
+        if (details.getData().getFinalOrder() == null
```

Comment:
Comments on the ticket suggests we also need to add logic for states related to Conditional Order

Reply:
Adam raised this on his review, Fubara agrees but still waiting on Kim to confirm so it can go in the ACs.